tests/test_wcsUtils.py

```diff
@@ -91,8 +91,8 @@ class ComputePixelToDistortedPixelTestCase(BaseTestCase):
     def testNoDistortion(self):
         """Test computePixelToDistortedPixel without distortion
 
-        Use an affine transform for pixelToFocalPlane; the transform
-        returned by computePixelToDistortedPixel should be the identity transform
+        Use an affine transform for pixelToFocalPlane; the transform returned
```

Comment:
What happened here?

Reply:
I miscalculated the number of characters and thought that line 95 was exceeding 79 characters. I'll drop this change and merge it to master.

Reply:
No, I was right the first time. Line 95 as it was exceeds 79 characters, and now it doesn't.